epictrack-web/src/components/shared/MasterTrackTable/utils.ts

```diff
@@ -72,17 +72,51 @@ interface ExportToCsvOptions<T extends MRT_RowData> {
   filenamePrefix: string;
 }
 
+function getStaffNamesByRole(row: any, roleName: string): string {
+  return row.staff
+    .filter(
+      (staffMember: { role: { name: string } }) =>
+        staffMember.role.name === roleName
+    )
+    .map(
+      (staffMember: { first_name: string; last_name: string }) =>
+        `${staffMember.last_name} ${staffMember.first_name}`
+    )
+    .join("; ");
+}
+
+const customAccessors: { [key: string]: (row: any) => any } = {
+  "Responsible EPD": (row) =>
+    `${row.responsible_epd?.first_name} ${row.responsible_epd?.last_name}`,
```

Comment:
This is a utils file for the MasterTrackTable. Why do we have mention of Responsible EPD here?

Reply:
lots of tables dont have accesor keys that are common and it causes issues with exporting

Reply:
but this component is general, it doesn't know or care about business logic like Responsible EPD or Work Lead. If this is the solution it shouldn't be here

Reply:
should i just allow passing custom accessors then into the export function